packages/victory-candlestick/src/victory-candlestick.tsx

```diff
@@ -40,7 +40,7 @@ export interface VictoryCandlestickStyleInterface {
 export type VictoryCandlestickLabelsType =
   | (string | number)[]
   | boolean
-  | ((datum: any) => number);
+  | ((datum: any) => number | string);
```

Comment:
@KenanYusuf Mind double checking this fix I put in? There was a TS error in the candlestick demo caused by this type not allowing the anon function to return string type. I assume that should be allowed as that's how it was being used in the demo.

Reply:
Yeah, this seems correct, good catch.

Reply:
Kewl, thanks for checking!